setup.py

```diff
@@ -54,11 +54,13 @@ def readfile(name):
 
 testing_extras = tests_require + ['coverage', 'nose']
 
-base_version = parse_version(VERSION).base_version
+major_version = ".".join(
+    str(v) for v in parse_version(VERSION)._version.release[:2]
+)
```

Comment:
Instead of using this private api I'd rather just use `'.'.join(VERSION.split('.')[:2])`.